CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ include(GNUInstallDirs)
 
 # Options
 set(MPAS_ALL_CORES atmosphere init_atmosphere)
-set(MPAS_CORES atmosphere CACHE STRING "MPAS cores to build. Options: ${MPAS_ALL_CORES}")
+set(MPAS_CORES init_atmosphere atmosphere CACHE STRING "MPAS cores to build. Options: ${MPAS_ALL_CORES}")
```

Comment:
I changed the default behavior to compiling both `init_atmosphere` and `atmosphere` cores.
I can revert this change if it is not desired.